src/Type.cc

```diff
@@ -45,7 +45,7 @@ const char* type_name(zeek::TypeTag t)
 		"port",      // 12
 		"addr",      // 13
 		"subnet",    // 14
-		"any",       // 15
+		"any",       // 16
```

Comment:
Still 15 for now.

Reply:
Gosh, counting is hard :). Thanks a lot.